src/canvas-extractor/edfi_canvas_extractor/client_facade.py

```diff
@@ -198,7 +198,7 @@ def extract_submissions(
         for assignment_id, submissions_df in submissions_dfs_by_assignment_id.items():
             section_id = str(section.id)
             submissions_df = submissionsMap.map_to_udm_submissions(submissions_df, section_id)
-            export[(section_id, str(assignment_id))] = submissions_df
+            export[(section_id, f"{section_id}-{str(assignment_id)}")] = submissions_df
```

Comment:
This causes the output directory to be like `assignment=104-111` instead of `assignment=111`. That's a good change, since the actual `SourceSystemId` is the two part value. Is it possible to get that value from the DataFrame instead of having to build it here?

Reply:
The issue with that approach is that here we are in the extraction part, not the mapping or the writing part, so we don't have access to the mapped SourceSystemIdentifier, and we need to build it here.

Reply:
Maybe we should move that code from the mapper here then, so that we can take this action only once. Something like

```python
assignment_source_system_identifier = f"{section_id}-{str(assignment_id)}"
submissions_df["assignment_id"] = assignment_source_system_identifier
```

Side note: in other cases, didn't we use a standard like `f"{section_id}#{str(assignment_id)}"` instead of `f"{section_id}-{str(assignment_id)}"`?

Reply:
I think so, maybe for section activity or something like that

Reply:
changes applied